src/tss2-fapi/ifapi_policy_callbacks.c

```diff
@@ -210,7 +210,11 @@ ifapi_get_object_name(
 
 /** Get public data of a NV object from keystore.
  *
- * @param[in] path The relative path of the NV object.
+ * @param[in] path The relative path of the NV object. The path will
+              be used to get the public from the keystore if no nv_index is
+              passed.
+ * @param[in] nv_index The index of the NV object. The index will be used to
+ *            get the public data if nv_index > 0.
```

Comment:
ahh ok if nv_index is 0 look at path..., is the inverse thru if not path look at index?

Reply:
> ahh ok if nv_index is 0 look at path..., is the inverse thru if not path look at index?

yes. It's ensured by `ifapi_json_TPMS_POLICYNV_deserialize` that either nv_index or path is set.